spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -6,9 +6,9 @@
   let(:contributor_user) { FactoryGirl.create(:user, username: "contributor_user", formal_name: "User, Contributor",  display_name: 'Contributor User') }
   let(:assistant) { FactoryGirl.create(:user, username: "ast9876") }
   let(:mesh_term) { "Vocabulary, Controlled" }
-  let(:expected_mesh_pid) { "D018875" }
+  let(:expected_mesh_id) { ::HeaderLookup::MESH_ID_URI + "D018875" }
```

Comment:
Just curious: why this syntax `::<class>::<attribute>` and not `<class>.<attribute>` in ruby?

Reply:
<class>.<attribute> is for calling class methods. ::<class>::<attribute> is specifically for getting a constant defined in a class.